projects/pgai/pgai/__init__.py

```diff
@@ -1 +1 @@
-__version__ = "0.4.0"
+__version__ = "0.1.0rc1"
```

Comment:
If we are going to change this (which we should), then please remove the `build_pgai_init_py` function from `build.py` too